pkg/api/handlers/compat/images.go

```diff
@@ -9,6 +9,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/sirupsen/logrus"
+
```

Comment:
No blank line

Reply:
The blank line was added by `goimports`.

Reply:
I could try deleting it, but don't know what linter would think about it.